Sources/Amplitude/Utilities/DefaultEventUtils.swift

```diff
@@ -29,11 +29,6 @@ public class DefaultEventUtils {
                     Constants.AMP_APP_PREVIOUS_VERSION_PROPERTY: previousVersion ?? "",
                 ])
             }
-            self.amplitude?.track(eventType: Constants.AMP_APPLICATION_OPENED_EVENT, eventProperties: [
```

Comment:
I'm curious if this was by design. Not sure if we have default charts for mobile but maybe check with code owner if you haven't already.

Reply:
This was a way to get around `applicationState` not being available to app extensions, but it's based on an invalid assumption that the app launch is correlated with an app open. The system will launch the app in the background in response to various background task completions.